main/telemetry.js

```diff
@@ -15,9 +20,23 @@ const writeClient = client.getWriteApi(
 )
 
 setInterval(() => {
-  writeClient.flush()
+  writeClient.flush().catch(console.error)
 }, 5000).unref()
 
+/**
+ * @param {Point} point
+ */
+const writePoint = point => {
```

Comment:
Is there any particular reason for choosing `=>` over `function` syntax?

```suggestion
function writePoint (point) {
```

I personally prefer `function` over `=>`, but I am happy to reconsider.

Reply:
I personally prefer arrow functions, since they can be shorter/more versatile, and V8 still can tell the function name:

```
> const foo = () => { throw new Error('oops') }
undefined
> foo()
Uncaught Error: oops
    at foo (REPL1:1:27)
```

I'm happy to change this to a `function` if you feel strongly about it.

Reply:
Fair enough 👍🏻